lib/src/main/java/io/github/taz03/jia/InstagramClient.java

```diff
@@ -49,20 +56,40 @@ public final class InstagramClient {
      * @param password The instagram password corresponding to the username provided
      */
     public InstagramClient(String username, String password) {
-        this(username, password, null);
+        this.username = username;
+        this.password = password;
+    }
+
+    public InstagramClient(String user,String pass,String id,String agent){
+        this.username = user;
+        this.password = pass;
+        this.userAgent = agent;
+        this.androidId = id;
+    }
+
+    private void configureDeviceIds() throws MalformedURLException, IOException {
+        var id = UUID.randomUUID().toString().replaceAll("-", "").toUpperCase();
+        this.androidId += "android-" + id.substring(id.length() - 16);
+        
+        var random = new Random().nextInt(1783);
+        var scanner = new Scanner(new URL("https://raw.githubusercontent.com/Taz03/JIA/main/user-agents.txt").openStream());
```

Comment:
instead put that in resources folder and read it from file directly

Reply:
> yeah but in this way, we can add or remove user agents directly from github

u can edit it directly from github as well if its in resources folder, changing the path won't disable the edit feature

> PS : Jio has blocked raw domain of github in india, putting it in the resourcy will be worthy right now

tf u mean, how is that related

Reply:
Which **resource** folder are you referring btw ?

Reply:
> Which resource folder are you referring btw ?

lib/src/main/resources

Reply:
It will increase the lib size unnecessarily. It's best to put it outside the build source